cypress/integration/T9.js

```diff
@@ -0,0 +1,87 @@
+import LoginPage from  '../objects/loginPage';
+import Task from '../objects/task';
+var login = new LoginPage();
+var task = new Task();
+
+
+describe('Sourcebooks testLogin', function(){
+    it('T1: TestLogin with Today date', function() {
+        login.visit();
+        login.userDropDown().click({force:true});
+        login.getUser("Dainius Gaidamavičius").click();
+        login.getRoleDropDown().click({force:true});
+        login.getRoleOption("Team Lead").click();
+        login.getSubmitButton().click();
+        login.getURL().should('include', '/time-logging');
+        login.getPageTitle().contains('Timesheets');
+        login.getCalendar().should('be.visible');
+        login.getTileForm().should('be.visible');
+        login.getUserTitle().contains('Dainius Gaidamavicius');
+        login.getMenuList().should('have.length', 2);          
+        login.getCalendarSpan().should('contain', new Date().getDate());
+    })
+})
+
+describe('Sorcebooks verify all user roles', function() {
+    let Roles = [
+        {
+            "name": "User",
+            "tabs": 1
+        },
+        {
+            "name": "Team Lead",
+            "tabs": 2
+        },
+        {
+            "name": "Manager",
+            "tabs": 5
+        },
+        {
+            "name": "Accountant",
+            "tabs": 5
+        }, 
+        {
+            "name": "Admin",
+            "tabs": 6
+        }];
+    
+    for(let i = 0; i < Roles.length; i++){
+        it('Verify that ' + Roles[i].name + ' can log in and should see ' + Roles[i].tabs + ' tabs', function(){
+        login.visit();
+        login.userDropDown().click({force:true});
+        login.getUser("Dainius Gaidamavičius").click();
+        login.getRoleDropDown().click({force:true});
+        login.getRoleOption(Roles[i].name).click();
+        login.getSubmitButton().click();
+        login.getUserTitle().contains('Dainius Gaidamavicius');
+        login.getMenuList().should('have.length', Roles[i].tabs);
+        login.getActiveMenuItem().should('have.css','color', 'rgb(64, 76, 237)');         
+        })
+    } 
+
+})
+
+describe('Sourcebooks creates task', function() {
```

Comment:
it would be a good idea to move this to new file, like integration/task.js

Reply:
I think you forgot to remove T3 from this file :)